druid.go

```diff
@@ -288,6 +288,14 @@ func (c *Client) Supervisor() *SupervisorService {
 	return &SupervisorService{client: c}
 }
 
+func (c *Client) Tasks() *TasksService {
+	return &TasksService{client: c}
+}
+
+func (c *Client) Metadata(waitTimeoutSeconds int, tickerMillis time.Duration) *MetadataService {
```

Comment:
Should we set the timing here. Timing parameters seem to be too specific for the await method so it feels unnecessary to request them here.

Wouldn't it make sense to have args in the await method? WDYT?

If we decide to have these parameters here then I suggest to use fuctional options pattern so that we have sane default and not force callers to figure out what to pass here even when they would not call the method.

WDYT?

Reply:
Thanks for the suggestion. I switched to functional options pattern with defaults.